package.json

```diff
@@ -12,7 +12,7 @@
     "test": "mocha --require @babel/register ajv/tests/*.js --exit",
     "lint": "npm run lint:schemas && npm run lint:test-schemas && npm run lint:ajv",
     "start": "DEBUG=ajv-schema-validator node ajv/app.js &",
-    "stop": "kill -9 $(lsof -ti:5002)",
+    "stop": "kill -9 $(lsof -ti:$PORT)",
```

Comment:
I get an error when running this: `lsof: unacceptable port specification in: -i :`